### PR TITLE
#1239 Make external lexer tab settings persistent.

### PR DESCRIPTION
**Pull request by sanastasiou**

### CODE REVIEW

File: PowerEditor/src/Parameters.cpp [hunk 1]
```diff
@@ -6173,10 +6174,21 @@ bool NppParameters::insertTabInfo(const TCHAR *langName, int tabInfo)
...
 		{
 			childNode->ToElement()->SetAttribute(TEXT("tabSettings"), tabInfo);
-			_pXmlDoc->SaveFile();
-			return true;
+			return _pXmlDoc->SaveFile();
 		}
 	}
+	auto aExternalLexerConfig = std::find_if(_pXmlExternalLexerDoc.begin(), _pXmlExternalLexerDoc.end(), [langName](TiXmlDocument * doc){
+		auto aLanguaneNode = (doc->FirstChild(TEXT("NotepadPlus")))->FirstChildElement(TEXT("Languages"))->FirstChildElement(TEXT("Language"));
+		auto *element = aLanguaneNode->ToElement();
```
Comment (xylographe) [inline PowerEditor/src/Parameters.cpp]: typo: `s/aLanguaneNode/aLanguageNode/`

Comment (sanastasiou) [inline PowerEditor/src/Parameters.cpp]: Hi are you talking about the tab or the local variables name?

Comment (xylographe) [inline PowerEditor/src/Parameters.cpp]: The local variable name "aLangua**n**eNode".
I assumed you ment "aLangua**g**eNode" (`g` i/o `n`).

Comment (sanastasiou) [inline PowerEditor/src/Parameters.cpp]: Damn I just saw this lol .. blind indeed...

I will "fix" this..

Comment (sanastasiou) [inline PowerEditor/src/Parameters.cpp]: Done...

**Comment by donho** [comment]
@sanastasiou  Sorry for the long wait.
Could you redo a PR? I promise it will be treated in the high priority if I can reproduce the bug and the PR provided fixes the bug.

**Pull request closed without merging**